receiver/README.md

```diff
@@ -95,22 +95,32 @@ receivers:
 
 ## <a name="jaeger"></a>Jaeger Receiver
 **Only traces are supported.**
-// TODO(ccaraman) Update Jaeger receiver documentation.
 
-This receiver receives spans from Jaeger collector HTTP and Thrift uploads and translates them into the internal span types that are then sent to the collector/exporters.
-Only traces are supported. This receiver does not support metrics.
+This receiver receives traces from [Jaeger](https://www.jaegertracing.io)
+instrumented applications. It translates them into the internal format and sends
+it to processors and receivers.
 
-Its address can be configured in the YAML configuration file under section "receivers", subsection "jaeger" and fields "collector_http_port", "collector_thrift_port".
-
-For example:
+It supports multiple protocols:
+- Thrift HTTP
+- Thrift TChannel
```

Comment:
Could you please file an item to remove TChannel support? For now this is correct, but we should remove it since it is legacy and If there is demand for it add it on contrib repo.

Reply:
https://github.com/open-telemetry/opentelemetry-service/issues/267 i'll add a todo in the jaeger receiver code base.

Reply:
Thanks!